README.ko

```diff
@@ -1,9 +1,10 @@
-=encoding utf8
+# vim: syntax=pod
 
 이 파일을 내용 그대로 읽고 있다면 우스꽝스러운 문자는 무시해주세요.
 이 문서는 POD로 읽을 수 있도록 POD 형식(F<pod/perlpod.pod> 문서를
 확인하세요)으로 작성되어 있습니다.
 
+=encoding utf8
```

Comment:
Why did you move this down? I think that's incorrect.

Reply:
@Leont you still have reservations on this? It makes sense to me. I will merge it if i dont hear from you soon to the contrary.

Reply:
No, it all makes sense to me now